### PR TITLE
Unify schema document and Schema Loader document

### PR DESCRIPTION
**Pull request by brfrn169**
This PR unifies `docs/schema.md` and `schema-loader/README.md` because there are some duplications in these documents. Please take a look!

### CODE REVIEW

File: schema-loader/README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Scalar DB Schema Loader
 
-Scalar DB Schema Loader creates, deletes and repairs Scalar DB schemas (namespaces and tables) on the basis of a provided schema file.
-Also, when you use the Consensus Commit transaction manager, it automatically adds the Scalar DB transaction metadata to the tables.
+Scalar DB has its own data model and schema, that maps to the implementation specific data model and schema.
+Also, it stores internal metadata (e.g., transaction ID, record version, transaction status) for managing transaction logs and statuses when you use the Consensus Commit transaction manager.
+It is a little hard for application developers to care for the schema mapping and metadata for transactions, so we offer a tool called Scalar DB Schema Loader for creating schema without much knowledge about those.
```
Comment (Torch3333) [inline schema-loader/README.md:5]:
```suggestion
It is a little hard for application developers to manage the schema mapping and metadata for transactions, so we offer a tool called Scalar DB Schema Loader for creating schema without requiring much knowledge about those.
```

File: schema-loader/README.md [hunk 2]
```diff
@@ -364,29 +365,35 @@ So, please set an appropriate value depending on the database implementations. P
...
+| TEXT     | VARCHAR(64)   | VARCHAR(10485760) | VARCHAR2(64) |
+| BLOB     | VARBINARY(64) |                   | RAW(64)      |
 
 If this data type mapping doesn't match your application, please alter the tables to change the data types after creating them with this tool.
 
+## Internal metadata for Consensus Commit
+
+The Consensus Commit transaction manager manages metadata (e.g., transaction ID, record version, transaction status) stored along with the actual records to handle transactions properly.
+Thus, along with any required columns by the application, additional columns for the metadata need to be defined in the schema.
+And this tool creates a table with the metadata when you use the Consensus Commit transaction manager.
```
Comment (Torch3333) [inline schema-loader/README.md:395]:
```suggestion
Additionaly, this tool creates a table with the metadata when you use the Consensus Commit transaction manager.
```

**Review by Torch3333 — COMMENTED** [review]
It looks good. Thank you. 
I left two suggestions. Please have a look when you can.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**